cohort-engine/src/main/java/com/ibm/cohort/engine/ClasspathLibrarySourceProvider.java

```diff
@@ -0,0 +1,29 @@
+package com.ibm.cohort.engine;
```

Comment:
No copyright?

Reply:
Missed a bunch of copyrights. Added in now.